### PR TITLE
v1.20.0-beta.1 changelog

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <krisss@vmware.com>

### CODE REVIEW

File: changelogs/CHANGELOG-v1.20.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+to any new object until an instance of Contour accepts the
+object and updates the status.
+
+(#4133, @stevesloka)
+
+## Source IP hash based load balancing
+
+Contour users can now configure their load balancing policies on `HTTPProxy` resources to hash the source IP of a client to ensure consistent routing to a backend service instance. Using this feature combined with header value hashing can implement advanced request routing and session affinity. Note that if you are using a load balancer to front your Envoy deployment, you will need to ensure it preserves client source IP addresses to ensure this feature is effective.
+
+See [this page](https://projectcontour.io/docs/v1.20.0/config/request-routing/#load-balancing-strategy) for more details on this feature.
```
Comment (sunjayBhatia) [inline changelogs/CHANGELOG-v1.20.0-beta.1.md:105]: ah, didnt anticipate a beta when writing this one, might need to adjust/remove this? i guess we won't have beta versioned docs

File: changelogs/CHANGELOG-v1.20.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+Now, if those weights are defined, they are applied so traffic is weighted appropriately across the services.
+Note that if no TCPProxy service weights are defined, traffic continues to be equally spread across all services.
+
+(#4169, @skriss)
+
+## Leader Election Configuration
+
+`contour serve` leader election configuration via config file has been deprecated.
+The preferred way to configure leader election parameters is now via command line flags.
+See [here](https://projectcontour.io/docs/v1.20.0/configuration/#serve-flags) for more detail on the new leader election flags.
```
Comment (sunjayBhatia) [inline changelogs/CHANGELOG-v1.20.0-beta.1.md:149]: similar issue here

File: changelogs/CHANGELOG-v1.20.0-beta.1.md (hunk 1 above)
Comment (sunjayBhatia) [reply changelogs/CHANGELOG-v1.20.0-beta.1.md:105]: could just adjust this instance to `main` instead of a version

Comment (skriss) [reply changelogs/CHANGELOG-v1.20.0-beta.1.md]: Good catch, I redirected to `main` docs.

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**